### ISSUE TITLE
Review localizable strings

### ISSUE THREAD

**Issue by mairaw** [dotnet/cli#5737]
I'd like to review them to make sure we're consistent on person used, punctuation, etc.

/cc @jonsequitur

**Comment by livarcocc** [comment]
@mairaw is this something that you still intend to do for 2.0? If so, please, touch base with @nguerrera

**Comment by mairaw** [comment]
@livarcocc I won't be able to work on this now. But I'd still would like to review the CLI text. I need to understand if it's the same the process or if it changed with the introduction of parser, etc.

**Comment by nguerrera** [comment]
The only difference now is that the strings in `*LocalizableStrings.cs` have been moved to `*LocalizableStrings.resx` since we are actually localized now. 

The parser has only a handful of strings of its own, but it has a mechanism to override them. On Monday, I'm going to use that, and all strings will be in this repo.

**Comment by mairaw** [comment]
Once that's done, we can plan the review then @nguerrera

**Comment by livarcocc** [comment]
@mairaw do you still intend to review these? Could it be done for the 2.1 timeframe?

**Comment by mairaw** [comment]
I actually had a draft email to you and @KathleenDollard about this that I started today. I think it’d be good but not sure if Kathleen wanted to look at this more closely beyond just basic grammar and punctuation.

**Comment by KathleenDollard** [comment]
I will not be able to work on this until after Tuesday. Does that push us beyond the 2.1 timeframe? 

I would love to have @mairaw review on this if you have the time.

**Comment by mairaw** [comment]
@livarcocc what's the deadline for getting 2.1 changes?

**Comment by livarcocc** [comment]
At this point, we probably missed the boat for 2.1.300, as I don't think this would meet the bar. Moving it out.

**Comment by mairaw** [comment]
Sounds good @livarcocc. I've been super swamped so this didn't make the cut for my priority list.

**Comment by livarcocc** [comment]
@mairaw @KathleenDollard is this something that you still intend to do? If so, do you have a timeline for it?

**Comment by KathleenDollard** [comment]
Do you have a link to the file(s) I should review? I can do this.

**Comment by mairaw** [comment]
Yes, we should do this, but I'm still slammed with other high-pri work. @KathleenDollard I know some of the files I'd review are under https://github.com/dotnet/cli/tree/master/src/dotnet/commands. Each command has a resources (resx) file and then you need to run the command as instructed [here](https://github.com/dotnet/cli/blob/master/Documentation/project-docs/developer-guide.md#buildingrunning) to update the xliff files too.

**Comment by KathleenDollard** [comment]
This is a very vague issue title, and I think that is making it difficult to complete. I would like to change this to "Review help text". I created a markdown file with all of our help in one file so that we can review in context. 

@dsplaisted can you tell me a place we can put the initial markdown file to ensure that after review we can compare with the original and see where changes are needed? I don't think we should keep this file, so if we could just add it to the repo as a WIP PR, get review, then kill the PR.

I would also like us to check top and bottom spacing to ensure consistency across help. I would like one line at the top and bottom, but I'm open on that. This issue or a separate issue. 

Similarly, what happens if you type `dotnet <commandname>` is not consistent.  I would prefer us not to display help, but I'm flexible if we are consistent. Technically that feels like a breaking change, but can't see it actually occurring and being a problem. Separate issue or this issue?

**Comment by dsplaisted** [comment]
To me it seems as if it depends on the command whether `dotnet <commandname>` is valid.  `dotnet run` works, but `dotnet tool` doesn't, so it makes sense for it just to display help.

As for where to put the markdown file, I would create a separate branch for it, possibly on your own fork of the repo, which we wouldn't expect to merge back into any other branches.  You could also probably just do it all in a PR that we wouldn't merge.

**Comment by KathleenDollard** [comment]
@dsplaisted on the display help on no argument question: When I last looked at it we had things that displayed errors and then did or did not display help. That's the inconsistency that is probably worth a look.

I got the branch/fork part. But was looking for a logical directory. Since it doesn't matter, I'll put it in the root. :-P

**Comment by mairaw** [comment]
Changed the title @KathleenDollard. I don't think the behavior should be part of this issue.
